docs/integrations/rust.md

```diff
@@ -0,0 +1,32 @@
+# Rust
+
+## Support
+
+Rust support relies on the presence of the `Cargo.lock` and `Cargo.toml` files.
+
+## Configuration
+
+### Automatic
+
+Run `fossa init` to detect all python directories that contain `Cargo.lock`.
+
+### Manual
+
+Add a module with `type: rust`, and `target` and `dir` set to the directory where `Cargo.lock` is.
+
+```yaml
+analyze:
+  modules:
+    - name: cargo-project
+      type: rust
+      target: rust/root
+      dir:  rust/root
+```
+
+## Analysis
+
+Rust analysis is a simple three step process:
+
+1. Parse `Cargo.lock` and retrieve information about the full dependency tree and each dependencies resolved version.
+2. Read the [root crate](https://doc.rust-lang.org/cargo/reference/manifest.html#the-workspace-section)'s `Cargo.toml` and detect if any "member" crates exist.
+3. Parse all `Cargo.toml` files to retrieve information about which dependencies the user declared.
```

Comment:
it's not entirely clear who "the user" is. I think "which dependencies are declared" is more precise